File: C/1012.c
#include <stdio.h>

int main ()
{
	  double A, B, C, pi, tri, cir, tra, qua, ret;
    pi =  3.14159;
	  scanf ("%lf %lf %lf", &A, &B, &C);
	  tri =  A  * C  / 2.0;
    cir = pi * (C * C);
    tra = (A + B) * C / 2.0;
    qua = B * B;
    ret = A * B;
  	printf ("TRIANGULO: %.3lf\n", tri);
    printf ("CIRCULO: %.3lf\n", cir);
    printf ("TRAPEZIO: %.3lf\n", tra);
    printf ("QUADRADO: %.3lf\n", qua);
    printf ("RETANGULO: %.3lf\n", ret);
    return 0;
}

File: Java/1012.java
import java.io.IOException;
import java.util.Scanner;
 
public class Main {
 
    public static void main(String[] args) throws IOException {
        double a,b,c,pi;
        pi = 3.14159;
        Scanner entrada = new Scanner (System.in);
        a = entrada.nextDouble();
        b = entrada.nextDouble();
        c = entrada.nextDouble();
        System.out.printf("TRIANGULO: %.3f\n", a  * c  / 2.0);
        System.out.printf("CIRCULO: %.3f\n", pi * (c * c));
        System.out.printf("TRAPEZIO: %.3f\n", (a + b) * c / 2.0);
        System.out.printf("QUADRADO: %.3f\n", b * b);
        System.out.printf("RETANGULO: %.3f\n", a * b);
    }
 
}
